game.js

```diff
@@ -28,17 +37,52 @@ const quitGame = () => {
  * @returns {undefined}
  */
 const gameLoop = () => {
+  console.log("I have a random number in mind")
+  console.log("It's between 1 and 1000");
+  console.log("You have 10 guesses total");
+  let randomNum = generateRandomNumber();
+  let guessNumber = rls.questionInt("Go ahead. Take a guess ")
+  let guessCount = 10;
+  
```

Comment:
You'll need a while loop here. For it to work. 
Calling gameLoop won't work because you'll always recreate the guessCount variable at 10.